app/src/main/java/com/asfoundation/wallet/viewmodel/TransactionsViewModel.java

```diff
@@ -406,6 +407,33 @@ public void showSettings(Context context) {
     transactionViewNavigator.openSettings(context, false);
   }
 
+  public void goToVipLink(Context context) {
+    Uri uri =
+        Uri.parse("https://appcoins.medium.com/welcome-to-the-appcoins-vip-program-7e5d133cc0ad");
+    transactionViewNavigator.navigateToBrowser(context, uri);
+  }
+
+  public MutableLiveData<Boolean> shouldShowVipBadge() {
```

Comment:
I know a lot of LiveDatas are being returned as MutableLiveData here, but it's wrong. I suspect people either didn't understand it or didn't care.

It's important to return LiveData instead of MutableLiveData... This is because no one should be able to change the value of these streams outside of ViewModel.